### ISSUE TITLE
Readme Usage Section

### ISSUE THREAD

**Issue by justkash** [simrat39/rust-tools.nvim#239]
I am new to using this plugin for Rust development and I was particularly interested in using the shortcuts to run specific tests under the cursor. However, I am unsure about how to achieve it.

The animated Gifs in the Readme Usage section are illustrative of the results that can be achieved with the tool, but the video is much too fast to learn from. It would be very beneficial to have a short description of what the video illustrates and a listing of the shortcuts used in the video itself.

I'd be happy to update the Readme myself, but I do not know where to find the documentation for the keymappings.

**Comment by simrat39** [comment]
You're right, the usage docs aren't great right now. I will definitely update the docs soon hopefully with better usage instructions
